Log file cannot be opened. Please check path. Trying to find:
/Users/rogerzhu/Documents/temoa/temoa-energysystem/data_files/debug_logs folder
Reading data files.[    0.47]
Creating Temoa model instance.[   20.34]
Solving.Exception found in solve_temoa_instance
[Errno 2] No such file or directory: '/Users/rogerzhu/Documents/temoa/temoa-energysystem/data_files/debug_logs/lp_files/tmp1wj19xpr.pyomo.lp'